src/.nuget/Microsoft.NETCore.Jit/descriptions.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "Name": "RuntimePackage",
+        "Description": "Internal implementation package not meant for direct consumption.  Please do not reference directly.",
+        "CommonTypes": [ ]
+    },
+    {
+        "Name": "Microsoft.NETCore.Jit",
```

Comment:
CoreCLR package is named as "Microsoft.NETCore.Runtime.CoreCLR". Should the JIT package be named something like "Microsoft.NETCore.CodeGen.JIT"?